account_asset_management/models/account_asset.py

```diff
@@ -296,20 +301,93 @@ def _compute_depreciation(self):
             depreciated = value_depreciated
             asset.update({"value_residual": residual, "value_depreciated": depreciated})
 
+    @api.depends("profile_id")
+    def _compute_group_ids(self):
+        for asset in self:
```

Comment:
Do we want to empty groups if no profile? If not, then put
```suggestion
        for asset in self.filtered('profile_id'):
```